From 0ff390a9809c54ec12a77892683e5f822fa14726 Mon Sep 17 00:00:00 2001
From: "Andre A. Gomes" <andremegafone@gmail.com>
Date: Thu, 21 Sep 2023 19:24:30 +0300
Subject: [PATCH 2/2] nyxt.asd: Add all systems defined by slynk and swank.

---
 nyxt.asd | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/nyxt.asd b/nyxt.asd
index baca244c4..f4f69460f 100644
--- a/nyxt.asd
+++ b/nyxt.asd
@@ -111,8 +111,16 @@ The renderer is configured from NYXT_RENDERER or `*nyxt-renderer*'."))
                clss
                spinneret
                slynk
+               slynk/arglists
+               slynk/fancy-inspector
+               slynk/indentation
+               slynk/mrepl
+               slynk/package-fu
+               slynk/stickers
+               slynk/trace-dialog
                sqlite
                swank
+               swank/exts
                trivia
                trivial-clipboard
                trivial-features
-- 
2.41.0

